tests/scripts/all.sh

```diff
@@ -3768,6 +3768,26 @@ component_test_aes_fewer_tables_and_rom_tables () {
     make test
 }
 
+component_test_aes_encrypt_only () {
+    msg "build: default config + PSA_CRYPTO_CONFIG - CIPHER_MODE_CBC - CIPHER_MODE_XTS - NIST_KW_C + AES_ENCRYPT_ONLY"
```

Comment:
This message is inaccurate - it doesn't explicitly set `MBEDTLS_AES_ENCRYPT_ONLY` ?